src/App.jsx

```diff
@@ -14,57 +15,59 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectProduct, setSelectProduct] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const GoodsList = ({ goodsList }) =>
+    goodsList.map(good => (
+      <tr
+        data-cy="Good"
+        className={good === selectProduct ? 'has-background-success-light' : ''}
+      >
+        <td>
+          <button
+            data-cy={good === selectProduct ? 'RemoveButton' : 'AddButton'}
+            type="button"
+            className={`button ${good === selectProduct ? 'is-info' : ''}`}
+            onClick={() => {
+              setSelectProduct(good === selectProduct ? null : good);
+            }}
```

Comment:
move it to the separate function 
```suggestion
            onClick={() => handleSelectGoods(good)}
```